domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -167,6 +169,7 @@ public String toString() {
     }
     spaceAppend(sb, limit.toString());
     spaceAppend(sb, offset.toString());
+    log.info("CQL >>> SQL " + this.query + " >>>" + sb.toString());
     return sb.toString();
   }
```

Comment:
The log entry

    CQL >>> SQL name==x >>> WHERE lower(f_unaccent(jsonb->>'name')) LIKE lower(f_unaccent('x'))

is more easy to understand with a colon:

    CQL >>> SQL: name==x >>> WHERE lower(f_unaccent(jsonb->>'name')) LIKE lower(f_unaccent('x'))